mycore-base/src/main/java/org/mycore/datamodel/ifs2/MCRIFS2Commands.java

```diff
@@ -187,6 +187,10 @@ private static Stream<FileInfo> getAllFiles(Path nodePath, Element node)
     public static void verifyVersioningMetadataStore(String storeId) {
         initMetadataStores();
         MCRVersioningMetadataStore store = MCRStoreCenter.instance().getStore(storeId);
+        if (storeId == null) {
```

Comment:
??? This does not make sense - it should be `if (store == null)`  ?

Reply:
ok